### ISSUE TITLE
[Request] Automaticly adding prefixes to the picture filenames in the chapter folder

### ISSUE THREAD

**Issue by Zirkulator** [NguyenDanPhuong/MangaRipper#78]
Hi!

**Thank you for your hard work!**
I have a question:
Is it possible to add a function that adds some prefixes (like the manga name, the title or at least the chapter number) to the (picture) filenames in the chapter folders
automaticly when downloading them?

For example:

0001.png       ->       <Manga Name>_<Chapter Nr.>_0001.png
0002.png       ->       <Manga Name>_<Chapter Nr.>_0002.png
0003.png       ->       <Manga Name>_<Chapter Nr.>_0003.png
...

Thank You!

**Comment by robert-7** [comment]
Yes please! Even something along the lines of having each picture saved with a `page` prefix. For example, even something like:

`a000.jpg -> [001] a000.jpg`
`a001.jpg -> [002] a001.jpg`
`a001b.jpg -> [003] a001b.jpg`
`a002.jpg -> [004] a002.jpg`
`dcredit.jpg -> [005] dcredit.jpg`

would be nice to have
